tests/slurm_plugin/test_clustermgtd.py

```diff
@@ -1274,47 +982,72 @@ def test_terminate_orphaned_instances(
+                DynamicNode("queue1-dy-c5xlarge-2", "ip-2", "hostname", "some_state", "queue1"),
             ],
-            [],
+            {
+                "queue1": SlurmPartition("queue1", "placeholder_nodes", "UP"),
+                "queue2": SlurmPartition("queue2", "placeholder_nodes", "INACTIVE"),
+                "queue3": SlurmPartition("queue3", "placeholder_nodes", "DRAIN"),
+                "queue4": SlurmPartition("queue4", "placeholder_nodes", "INACTIVE"),
+            },
         ),
         (
             True,
             False,
             [EC2Instance("id-1", "ip-1", "hostname", "launch_time")],
             [
-                SlurmNode("queue1-st-c5xlarge-1", "ip", "hostname", "some_state", "queue1"),
-                SlurmNode("queue1-st-c5xlarge-2", "ip", "hostname", "some_state", "queue1"),
+                StaticNode("queue1-st-c5xlarge-1", "ip", "hostname", "some_state", "queue1"),
+                DynamicNode("queue1-dy-c5xlarge-2", "ip", "hostname", "some_state", "queue1"),
             ],
-            [],
+            {
+                "queue1": SlurmPartition("queue1", "placeholder_nodes", "UP"),
+                "queue2": SlurmPartition("queue2", "placeholder_nodes", "INACTIVE"),
+                "queue3": SlurmPartition("queue3", "placeholder_nodes", "DRAIN"),
+                "queue4": SlurmPartition("queue4", "placeholder_nodes", "INACTIVE"),
+            },
         ),
         (
             False,
             True,
             [EC2Instance("id-1", "ip-1", "hostname", "launch_time")],
             [
-                SlurmNode("queue1-st-c5xlarge-1", "ip", "hostname", "some_state", "queue1"),
-                SlurmNode("queue1-st-c5xlarge-2", "ip", "hostname", "some_state", "queue1"),
+                StaticNode("queue1-st-c5xlarge-1", "ip", "hostname", "some_state", "queue1"),
+                DynamicNode("queue1-dy-c5xlarge-2", "ip", "hostname", "some_state", "queue1"),
             ],
-            [],
+            {
+                "queue1": SlurmPartition("queue1", "placeholder_nodes", "UP"),
+                "queue2": SlurmPartition("queue2", "placeholder_nodes", "INACTIVE"),
+                "queue3": SlurmPartition("queue3", "placeholder_nodes", "DRAIN"),
+                "queue4": SlurmPartition("queue4", "placeholder_nodes", "INACTIVE"),
+            },
         ),
         (
             False,
             True,
             [EC2Instance("id-1", "ip-1", "hostname", "launch_time")],
-            [],
             [
-                SlurmNode("inactive-queue1-st-c5xlarge-1", "ip", "hostname", "some_state", "inactive-queue1"),
-                SlurmNode("inactive-queue1-st-c5xlarge-2", "ip", "hostname", "some_state", "inactive-queue1"),
+                StaticNode("inactive-queue1-st-c5xlarge-1", "ip", "hostname", "some_state", "inactive-queue1"),
+                DynamicNode("inactive-queue1-dyt-c5xlarge-2", "ip", "hostname", "some_state", "inactive-queue1"),
```

Comment:
Yes, it is a typo. Fixed. It doesn't matter for the test because  it is the name of Slurm node, we use the information to initialize node to be DynamicNode or StaticNode. Since we already know it is a dynamic node or static node from its class, it is not used here